Stylesheets/Components/rating.css

```diff
@@ -0,0 +1,25 @@
+.ratings{
+    display: flex;
+    justify-content: space-around;
+}
+
+.star{
```

Comment:
You can give class names specific to components. i.e. `rating-start`.
This will not raise conflicts in future projects.

Reply:
changed thanks